multigeiger/chkhardware.cpp

```diff
@@ -0,0 +1,19 @@
+// Check if we have LoRa hardware
+
+// Test pin 26 (that is LORA_DIO0) to be low.
+#include <Arduino.h>
+
+#define HWTESTPIN 26
+
+static bool haveLoRa = false;
```

Comment:
maybe this should be rather `board(_type)` (not "CPU" though as it was in userdefines.h, but with the same semantics as we had there: one specific integer identifying a specific board).

Reply:
see comment above